apps/masterbots.ai/components/layout/sidebar/sidebar-link.tsx

```
@@ -161,14 +172,22 @@ const ChatbotComponent: React.FC<ChatbotComponentProps> = React.memo(function Ch
   isFilterMode,
   page
 }) {
-  const { selectedChatbots, toggleChatbotSelection } = useSidebar()
+  const { selectedChatbots, toggleChatbotSelection,navigateTo } = useSidebar()
   const pathname = usePathname()
   const isBrowse = !pathname.includes('/c') && !pathname.includes('/u')
   const { slug } = useParams()
 
   const handleChatbotClick = useCallback((e: React.MouseEvent) => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: category.name.toLowerCase

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)